src/encoded/batch_download.py

```diff
@@ -462,9 +487,13 @@ def batch_download(context, request):
     default_params = [
         ('limit', 'all'),
         ('field', 'files.href'),
-        ('field', 'files.restricted')
+        ('field', 'files.restricted'),
+        ('field', 'files.file_format'),
+        ('field', 'files.file_format_type'),
+        ('field', 'files.status'),
     ]
     qs = QueryString(request)
+    param_list = qs.group_values_by_key()
```

Comment:
Looks like you could avoid this call if you did `visualizable_only = qs.is_param('visualizable', 'true')` instead.